### ISSUE TITLE
Bang Dream Girls Band Party for Nintendo Switch

### ISSUE THREAD

**Issue by ToshiyukiDoma** [Ryujinx/Ryujinx-Games-List#3720]
#### Current on `master` : 1.0.7038

The game just crashes after selecting "Tap Any Button to start."

```
Last error returned.
Unhandled exception caught: LibHac.HorizonResultException: ResultFsNonRealDataVerificationFailed (2002-4604): Hash error!
```

#### Pull requests needed :

*Add needed PR links here.*

*(Optional section, remove it if unneeded)*

#### Outstanding Issues:

*Add issue links here.*

*Example: https://github.com/Ryujinx/Ryujinx/issues/xxx (xxx = number issue)*

*(Optional section, remove it if unneeded)*

#### Screenshots :

![image](https://user-images.githubusercontent.com/23119677/133785682-40f0e862-3bd8-4190-a0a6-f9a00a8eb89c.png)

*(Optional section, remove it if unneeded)*

#### Log file :

[Ryujinx_1.0.7038_2021-09-17_20-50-39.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/7185752/Ryujinx_1.0.7038_2021-09-17_20-50-39.log)

**Comment by vojo915** [comment]
**Game Update Version :** 1.0.1  
**Current on `master` :** 1.0.7038

**Required:** Sound = **OpenAL**
\* **SoundIO** causes stutter.

As long as you play it on a regular gamepad, it works fine.

\* Touch operation and Joy-Con (gyro sensor) operation cannot be tested due to lack of environment.

**Hardware Specs :**  
**CPU:** Ryzen 5 3600  
**GPU:** NVIDIA GeForce GTX 1660 Ti  
**RAM:** 16GB

**Screenshots:**

![1_q99](https://user-images.githubusercontent.com/84438932/133877934-d184d948-b16b-46f2-91a6-622615de1f7d.jpg)
![2_q99](https://user-images.githubusercontent.com/84438932/133877940-68d72095-fcf0-4c3c-8f80-64ce97b5504e.jpg)
![3_q99](https://user-images.githubusercontent.com/84438932/133877946-c352145b-4e47-4987-9700-2cad8df845a4.jpg)
![4_q99](https://user-images.githubusercontent.com/84438932/133877952-b412a718-03d3-4ded-ae80-5b7ae2ab56da.jpg)
![5_q99](https://user-images.githubusercontent.com/84438932/133877960-28c31c06-3965-4961-bc52-83c45ca72ad9.jpg)
![6_q99](https://user-images.githubusercontent.com/84438932/133877966-eaf8ac44-de63-419c-bd5b-e9001fc9bc5d.jpg)
![7_q99](https://user-images.githubusercontent.com/84438932/133877977-243ad30c-eebe-4ebc-9535-6870ca027e61.jpg)
![8_q99](https://user-images.githubusercontent.com/84438932/133877981-ad15527f-bb8c-4640-b31c-c4dac3e0ea95.jpg)
![9_q99](https://user-images.githubusercontent.com/84438932/133877985-b24049af-cd42-42a8-aa55-57b4e975c99e.jpg)
![10_q99](https://user-images.githubusercontent.com/84438932/133877990-9ed59f56-a8c7-47b6-a2fe-75196b940993.jpg)

**Log File:**

[Ryujinx_1.0.7038_010070D01483A000.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/7189340/Ryujinx_1.0.7038_010070D01483A000.log)

**Comment by ToshiyukiDoma** [comment]
Yes. I actually had to try it again and found out my copy is damaged. It worked well now. Going to close this.

**Comment by gdkchan** [comment]
The issue should remain open, we use it to keep track of game state (even if it is already fully functional). I will edit the first post with the more complete test.
